ui/narrative/methods/annotate_metagenome/spec.json

```diff
@@ -7,7 +7,7 @@
     "categories": ["active", "annotation"],
     "widgets": {
         "input": "kbaseNarrativeMethodInput",
-        "output": "kbaseAnnotatedMetagenomeAssemblyView"
+        "output": "no-display"
```

Comment:
How come we are removing the AMA output?

Reply:
its not functional at the moment, and it was specifically requested to be removed by paramvir and dylan